build/targets/KoreBuild.Common.props

```diff
@@ -1,6 +1,12 @@
 <Project>
   <PropertyGroup>
     <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+    <!--
+      The environment variable 'CI' is set to 'true' by Travis and AppVeyor.
+      https://docs.travis-ci.com/user/environment-variables/
+      https://www.appveyor.com/docs/environment-variables/
+    -->
+    <Configuration Condition="'$(Configuration)' == '' AND '$(CI)' == 'true'">Release</Configuration>
```

Comment:
Should be above the previous line. Configuration remains `Debug` as you've got it.